cirq-google/cirq_google/workflow/qubit_placement_test.py

```diff
@@ -22,6 +22,12 @@
 class TestDevice(cirq.Device):
     def __init__(self):
         self.qubits = cirq.GridQubit.rect(2, 8)
+        neighbs = [(a, b) for a in self.qubits for b in self.qubits if a.is_adjacent(b)]
```

Comment:
Nit: spare the two extra characters to make this read nicely?

Reply:
Done.